packages/flutter/lib/src/rendering/layer.dart

```diff
@@ -168,6 +168,15 @@ abstract class Layer extends AbstractNode with DiagnosticableTreeMixin {
 
   bool _debugMutationsLocked = false;
 
+  /// Whether or not this layer, or any child layers, can be rasterized with
+  /// [Scene.toImage] or [Scene.toImageSync].
+  ///
+  /// If `false`, calling the above methods may yield an image which is
```

Comment:
Document: Is what this returns allowed to change during the lifetime of the Layer or must it stay static? If it can change, do you need to notify anybody if it does?

Reply:
Added comment that this value is assumed to not change

Reply:
Now that I read over this again I think that statement is actually not true, right? Layers get reused and if at one point one of your children is a platform view, this will return false, if then that platform view goes away, this will be true. So, it does change...

Reply:
Oh ... right. The value may change. I think I was thinking about individual layers, but then it actually doesn't really matter. You only generate a snapshot for a specific instance of a layer tree.